src/Ocelot/Configuration/Creator/RoutesCreator.cs

```diff
@@ -21,8 +21,10 @@ public class RoutesCreator : IRoutesCreator
         private readonly IRouteKeyCreator _routeKeyCreator;
         private readonly ISecurityOptionsCreator _securityOptionsCreator;
         private readonly IVersionCreator _versionCreator;
-        private readonly IVersionPolicyCreator _versionPolicyCreator;
-        private readonly IMetadataCreator _metadataCreator;
+        private readonly IVersionPolicyCreator _versionPolicyCreator;
+        private readonly IMetadataCreator _metadataCreator;
+        
+        public const int DefaultRequestTimeoutSeconds = 90;

```

Comment:
to go all the way, this parameter should be a setting in globalConfiguration

Reply:
Good proposal!
Good news that we already have global value in current settings:
https://github.com/ThreeMammals/Ocelot/blob/cc8f5c5dfc7c56e4446a2e80d97391429c8b364d/src/Ocelot/Configuration/File/FileGlobalConfiguration.cs#L24
But we don't use it! 🤯

Reply:
@RaynaldM We have [this](https://github.com/ThreeMammals/Ocelot/pull/2073/files#diff-95ae956905b7ba73232d0c91aaccc4f723aaa9eeb879210e718b62e43b847ca6R26) as global timeout value.
This 90sec constant is used when both route and global values are not defined!